assembl/static2/js/app/utils/timeline.js

```diff
@@ -24,27 +25,32 @@ const dateComparator = (phase1, phase2) => {
   return 1;
 };
 
-export const getCurrentPhaseIdentifier = (_timeline: Timeline) => {
+export const getCurrentPhase = (_timeline: Timeline) => {
   let timeline = _timeline;
   if (!timeline) {
     timeline = [];
   }
 
   const currentDate = new Date();
-  let identifier = '';
+  let currentPhase = null;
   timeline.forEach((phase) => {
     const startDate = new Date(phase.start);
     const endDate = new Date(phase.end);
     if (currentDate >= startDate && currentDate < endDate) {
-      identifier = phase.identifier;
+      currentPhase = phase;
     }
   });
-  if (identifier) {
-    return identifier;
+  if (currentPhase) {
+    return currentPhase;
   }
   // if all phases are closed, take the last closed phase
   const sortedTimeline = [...timeline].sort(dateComparator);
-  return sortedTimeline.length === 0 ? 'thread' : sortedTimeline[sortedTimeline.length - 1].identifier;
+  return sortedTimeline[sortedTimeline.length - 1];
+};
+
+export const getCurrentPhaseIdentifier = (_timeline: Timeline) => {
+  const currentPhase = getCurrentPhase(_timeline);
+  return currentPhase ? currentPhase.identifier : PHASES.thread;
```

Comment:
Remove the `getCurrentPhaseIdentifier` function completely if it's not used in the code base anymore.

Like I said in a comment above, you may consider having a function
```
export const getCurrentPhaseData = (_timeline: Timeline) => {
￼   const currentPhase = getCurrentPhase(_timeline);
￼   return currentPhase ? {phase: currentPhase.identifier, phaseId: currentPhase.id} : {phase: PHASES.thread, phaseId: null};
```